### PR TITLE
Comment-out compliance dashboard until further notice.

### PR DESCRIPTION
**Pull request by afwillia**
This PR will disable the compliance dashboard. We won't use that dashboard in the main release of the app until further notice.

### CODE REVIEW

File: ui.R [hunk 1]
```diff
@@ -121,8 +121,8 @@ ui <- shinydashboardPlus::dashboardPage(
               label = "Data Type:",
               choices = "Generating..."
             )
-          ),
-          dashboardUI("dashboard")
+          )#,
```
Comment (rrchai) [inline ui.R:124]: is that '#' typo?

**Comment by milen-sage** [comment]
I'd defer my review to @rrchai :)

**Review by rrchai — CHANGES_REQUESTED** [review]
LGTM, just one minor change.

Additionally, you could also ignore loading the dashboard related modules/functions to speed up app initialization by add `%>% .[!grepl("dashboard", .)]` to the end of this line:
https://github.com/Sage-Bionetworks/data_curator/blob/e13593c27378f4c02db1d0d448ab57c45c4f53c2/global.R#L109.

File: ui.R (hunk 1 above)
Comment (rrchai) [reply ui.R:124]: Oh, I see this '#' is used to comment out the ",".

**Review by rrchai — APPROVED** [review]

**Comment by afwillia** [comment]
Thanks @rrchai good idea!

**Pull request merged**